app/controllers/UserController.scala

```diff
@@ -61,8 +63,9 @@ class UserController @Inject()(val messagesApi: MessagesApi)
 
   def userLoggedTime(userId: String) = SecuredAction.async { implicit request =>
     for {
-      user <- UserDAO.findOneById(userId) ?~> Messages("user.notFound")
-      _ <- user.isEditableBy(request.identity) ?~> Messages("notAllowed")
```

Comment:
How was `isEditableBy` actually checked? Did this happen via the `?~>`? I thought that was only checking that the option is defined, but not the packed boolean.

Reply:
good catch, needs `Fox.assertTrue`. The ?~> just adds the failure message

Reply:
I wasn't aware of the implicit conversion, see #2941.

Let's double check all the changes where `user.isEditableBy` was used before, if they have `assertBoolean`/`assertTrue`.